packages/create-llama/helpers/is-folder-empty.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import fs from "fs";
+import path from "path";
+import { blue, green } from "picocolors";
+
+export function isFolderEmpty(root: string, name: string): boolean {
+  const validFiles = [
+    ".DS_Store",
+    ".git",
+    ".gitattributes",
+    ".gitignore",
+    ".gitlab-ci.yml",
+    ".hg",
+    ".hgcheck",
+    ".hgignore",
+    ".idea",
```

Comment:
add `.vscode` ?

Reply:
helpers are from https://github.com/vercel/next.js/tree/v13.5.6/packages/create-next-app/helpers so I wouldn't touch them if not necessary. Then we can just copy them with new releases.